src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -2,159 +2,262 @@
+
+```
+How can I improve my alerts setup?
+```
 
-* How can I use Terraform to create dashboards? 
-* How can I improve my alerts setup?
-* How can I instrument AWS?
+```
+How can I instrument AWS?
+```
     </TabsPageItem>
 
     <TabsPageItem id="2">
     If you want to get to the root cause of latency issues for your app, here are some good questions to ask:
 
-* What are some recent incidents in my system?
-* Which infrastructure hosts are showing signs of degradation over the past hour?
+```
+What are some recent incidents in my system?
+```
+
+```
+Which infrastructure hosts are showing signs of degradation over the past hour?
+```
     </TabsPageItem>
 
     <TabsPageItem id="3">
-    Our AI can also be your partner in observability maturity. If you're responsible for ensuring that your app can handle spikes for a big e-commerce event like Black Friday, it's important to understand the state of your system so you can improve it _ahead of time_. Ask our AI:
+New Relic AI can also be your partner in observability maturity. If you're responsible for ensuring that your app can handle planned spikes, such as a big e-commerce sale, it's essential to understand the state of your system so you can improve it _ahead of time_.  Try asking New Relic AI:
+
+```
+How can I improve the performance of my infrastructure?
+```
 
-* How can I improve the performance of my infrastructure?
-* Were unique user sessions up or down last week compared to the week before?
+```
+Were unique user sessions up or down last week compared to the week before?
+```
     </TabsPageItem>
     </TabsPages>
-</Tabs>
+    </Tabs>
 
-### Get help creating NRQL queries [#NRQL-console]
+## Create NRQL queries [#NRQL-console]
 
-[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool you can use to gain insights into systems, but when you're in a hurry you might not have time to create the right query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
+[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool to gain insights into your systems. Still, when you're in a hurry, you might not have time to create the correct NRQL query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. 
 
 <img
-        title="Screenshot showing the Ask AI button in the New Relic platform"
-        alt="ask AI button"
-        src={askgrok}
-    />
-
-
-To enable NRAI in the NRQL console: 
+    title="New Relic AI NRQL "
```

Comment:
@homelessbirds to get to the image shown here, did users click the **Ask AI** button at the top right and ask for help with NRQL or did they click **Query using AI - Experimental**?